sunpy/net/base_client.py

```diff
@@ -39,9 +39,39 @@ def search(self, *args, **kwargs):
         """
 
     @abstractmethod
-    def fetch(self, *args, **kwargs):
+    def fetch(self, *query_results, path=None, overwrite=False, progress=True,
```

Comment:
Now that it defaults to not overwrite - can the progress bar not appear if file already exists and it should maybe print 'file exists /path' ?

Reply:
The main progress bar appears as it processes each file (it has to ask the server what the filename is before it can determine if it exists or not). The sub progress bars only appear for files that are actually downloading.

With regards to saying they already exist, I don't want to print out as it would mess with the progress bar, I am not really sure how I would tell the end user that the files existed already at all. What do you think the value of doing that is? (so I can work out what the best way to do it would be?)